app/database/models/server/team.ts

```diff
@@ -108,13 +108,21 @@ export default class TeamModel extends Model implements TeamModelInterface {
             Q.where('team_id', this.id),
             Q.where('loadedInGlobalThreads', true),
         )),
+        Q.and(
+            Q.where('reply_count', Q.gt(0)),
+            Q.where('is_following', true),
+        ),
         Q.sortBy('last_reply_at', Q.desc),
     );
 
     /** unreadThreadsList : Unread threads list belonging to a team */
     @lazy unreadThreadsList = this.collections.get<ThreadModel>(THREAD).query(
         Q.on(THREADS_IN_TEAM, 'team_id', this.id),
-        Q.where('unread_replies', Q.gt(0)),
+        Q.and(
```

Comment:
This intrigues me and I would like to see what the `unreadThreadsList` looks like.  Can you please share a small log of it?

Reply:
<img width="596" alt="Screenshot 2022-04-06 at 4 33 24 PM" src="https://user-images.githubusercontent.com/3829551/161988511-c0a3f038-b5eb-4122-9b50-2da475dd14ea.png">

don't mind the `unread_replies/mentions` numbers 😅

Reply:
usage would be: `team.threadsList.observe()`

Reply:
> don't mind the `unread_replies/mentions` numbers 😅

Fixing that as well in this PR

Reply:
It looks good.  

This is not blocking but a point for discussion, if I recall correctly, the ThreadsInTeam table was created to facilitate pagination.  But seeing this lazy query, now makes me question if the 'Threads' table would have been enough if we added the extra fields from 'ThreadsInTeam' into it.  What do you think ?